Mathlib/Tactic/DeclarationNames.lean

```diff
@@ -47,3 +47,9 @@ def getAliasSyntax {m} [Monad m] [MonadResolveName m] (stx : Syntax) : m (Array
       aliases := aliases.push
         (mkIdentFrom (.ofRange (idStx.raw.getRange?.getD default)) (currNamespace ++ id))
   return aliases
+
+/-- Used for linters which use `0` instead of `false` for disabling. -/
+def logLint0Disable {m} [Monad m] [MonadLog m] [AddMessageContext m] [MonadOptions m]
+    (linterOption : Lean.Option Nat) (stx : Syntax) (msg : MessageData) : m Unit :=
```

Comment:
Should this take just a linter name - or are you intentionally passing an `Option` for future flexibility?